package.json

```diff
@@ -29,7 +29,6 @@
     "example": "examples"
   },
   "scripts": {
-    "postinstall": "[ \"$NODE_ENV\" != production ] && npm ln ../jsdom && npm ln ../buble-tape",
```

Comment:
Nope. Problem solved earlier this week @brandondees 

/cc @mrbernnz may want to remove this commit out of this branch with a rebase